src/bitmap/proptests.rs

```diff
@@ -528,7 +528,11 @@ mod test {
             prop_assert_eq!(&a, &b);
             prop_assert!(a.is_subset(&b) && b.is_subset(&a));
 
-            b.insert(a.max().unwrap_or(0) + 1);
```

Comment:
This will panic if `max() == u32::MAX`

Reply:
You are absolutely right!

Reply:
I know. proptest told me so. In big red letters.